x/ibc/applications/transfer/module.go

```diff
@@ -193,6 +194,14 @@ func (am AppModule) OnChanOpenInit(
 	counterparty channeltypes.Counterparty,
 	version string,
 ) error {
+	// NOTE: for escrow address security only 2^32 channels are allowed to be created
```

Comment:
could you link to the discussion here: https://github.com/cosmos/cosmos-sdk/issues/7737

Reply:
good idea :+1: